kolibri/plugins/coach/assets/src/views/create-exam-page/index.vue

```diff
@@ -25,21 +25,74 @@
       @input="validateNum = true"
     />
 
-    <div v-if="channelExercises">
-      <h2>Choose exercises</h2>
-      <textbox
-        :ariaLabel="$tr('searchContent')"
-        :placeholder="$tr('searchContent')"
-        v-model.trim="searchInput"
-      />
-      <div v-if="searchInput">
-        search results
-      </div>
-      <div v-else>
-        breadcrumbs
-        topic list
+    <h2>Choose exercises</h2>
+    <textbox
+      :ariaLabel="$tr('searchContent')"
+      :placeholder="$tr('searchContent')"
+      v-model.trim="searchInput"
+    />
+    <div v-if="searchInput">
+      search results
+    </div>
+    <div v-else>
+      <nav>
+        <ol>
+          <li v-for="(topic, index) in topic.breadcrumbs" :class="[notLastBreadcrumb(index) ? 'not-last' : '']">
```

Comment:
Maybe make your method return the class name string so that you don't have as much logic here